Build failed in Jenkins: SONiC VPP VM » Build-VM-Image #69

Build failure: http://172.22.64.183:8080/job/SONiC-VPP-VM/job/Build-VM-Image/69

As per the logs the issue is seen while building target p4rt , which is a google specific network controller agent.
 
/usr/share/depot_tools/vpython3: line 48: /usr/share/depot_tools/.cipd_bin/vpython3: No such file or directory
/usr/share/depot_tools/vpython3: line 48: /usr/share/depot_tools/.cipd_bin/vpython3: No such file or directory
make[1]: *** [Makefile:34: /sonic/target/debs/bullseye/sonic-p4rt_0.0.1_amd64.deb] Error 127

Seem like fix PR  https://github.com/sonic-net/sonic-buildimage/pull/16114 is merged to master.   Feel free to change label when you get a chance.
 
 
From: "Shashidhar Patil (shaship)" <[shaship@cisco.com](mailto:shaship@cisco.com)>
Date: Wednesday, August 16, 2023 at 10:52 PM
To: "Nikunj Desai (nsdesai)" <[nsdesai@cisco.com](mailto:nsdesai@cisco.com)>
Subject: Re: Build failed in Jenkins: SONiC VPP VM » Build-VM-Image #64
 
It is quite surprising that my build went through, azure went through. I see that the culprit code exists in  the label we are using.
It looks like some build environment issue. Anyway, lets wait for a latest successful label .
 
Thanks for pointing it out.
 
-Shashidhar
 
From: Shashidhar Patil (shaship) <[shaship@cisco.com](mailto:shaship@cisco.com)>
Date: Thursday, 17 August 2023 at 9:56 AM
To: Nikunj Desai (nsdesai) <[nsdesai@cisco.com](mailto:nsdesai@cisco.com)>
Subject: Re: Build failed in Jenkins: SONiC VPP VM » Build-VM-Image #64

The WORKING_LABEL in sonic-vpp-set.sh gives the label used. We will wait for the mainline to get the fix merged before updating the label.
 
Thanks
-Shashidhar
 
From: Nikunj Desai (nsdesai) <[nsdesai@cisco.com](mailto:nsdesai@cisco.com)>
Date: Thursday, 17 August 2023 at 12:41 AM
To: Shashidhar Patil (shaship) <[shaship@cisco.com](mailto:shaship@cisco.com)>
Subject: Re: Build failed in Jenkins: SONiC VPP VM » Build-VM-Image #64

In my workspace, I made changes to ‘sonic-slave-bullseye/Dockerfile.j2’ suggested in PR. Now, VM build is successful.
 
Once this PR is merged, I believe build should go through fine as we are getting latest code from sonic-buildimage.
 
Thanks,
Nikunj
 
From: "Nikunj Desai (nsdesai)" <[nsdesai@cisco.com](mailto:nsdesai@cisco.com)>
Date: Wednesday, August 16, 2023 at 10:47 AM
To: "Shashidhar Patil (shaship)" <[shaship@cisco.com](mailto:shaship@cisco.com)>
Subject: Re: Build failed in Jenkins: SONiC VPP VM » Build-VM-Image #64
 
VM build is still failing for me.  Came across following issue reporting same build error for master and 202305 branch.
 
https://github.com/sonic-net/sonic-buildimage/issues/16087
 
Fix - https://github.com/sonic-net/sonic-buildimage/pull/16114 is not merged yet.

--- a/sonic-vpp-setup.sh
+++ b/sonic-vpp-setup.sh
@@ -26,8 +26,8 @@ cd ./build/sonic-buildimage
 
 # Below is the build label information
 
-#327510	20230731.7	master	Azure.sonic-buildimage.official.vs	succeeded	2023-07-31T08:36:27	2023-07-31T13:56:49	141132c94b
-WORKING_LABEL=141132c94b
+#348554 20230828.7      master  Azure.sonic-buildimage.official.vs      succeeded       2023-08-28T08:07:21     2023-08-28T12:54:41     c8465c0d9a
+WORKING_LABEL=c8465c0d9a
 
 SONIC_CHECKOUT_LABEL=${SONIC_CHECKOUT_LABEL:=$WORKING_LABEL}
 

